include/language.php

```diff
@@ -432,8 +620,18 @@ public function get_display_flag_url() {
 	 * @return void
 	 */
 	public function update_count() {
-		wp_update_term_count( $this->term_taxonomy_id, 'language' ); // Posts count.
-		wp_update_term_count( $this->tl_term_taxonomy_id, 'term_language' ); // Terms count.
+		foreach ( $this->term_props as $taxonomy => $props ) {
+			wp_update_term_count( $props['term_taxonomy_id'], $taxonomy );
+		}
+
+		/**
+		 * Fires when updating language term counts.
+		 *
+		 * @since 3.4
+		 *
+		 * @param PLL_Language $language The language object.
+		 */
+		do_action( 'pll_updated_language_term_counts', $this );
```

Comment:
Why this action? Do you plan to use it elsewhere? Or is it for our users?

Reply:
It's a hook I added at first, but we don't need it anymore.
I removed it in [47b6a57](https://github.com/polylang/polylang/pull/1149/commits/47b6a578803e5c072fb6f30eec7d9a6dfe19a188).